### DISCUSSION TITLE
Is it possible to have two different endpoints?

### DISCUSSION THREAD

**Discussion by smolinari** [mercurius-js/mercurius#623] [Q&A] [answered]
Hi,

I'm wondering if it is possible to have two different endpoints. I'd like to have a specialized endpoint for refreshing tokens. 

Scott

**Comment by mcollina** [comment, marked as answer]
You can register the plugin twice:

```js
app.register(async function (app) {
  app.register(mercurius, { ... })
}, { prefix: '/a' })
app.register(async function (app) {
   app.register(mercurius, { ... })
}, { prefix: '/b' })
```

**Reply by smolinari** [reply]
Whoa. Is it that simple? I'm also working with NestJS and this wrapper module. 

https://github.com/Davide-Gheri/nestjs-mercurius

I guess I'll have to see if it allows for two root Graphql modules. 

Scott

**Reply by mcollina** [reply]
> Whoa. Is it that simple? 

Yes it is. I don't think this capability is surfaced to Nest.
